fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -1291,6 +1291,10 @@ public PlanFragment visitPhysicalHashAggregate(OptExpression optExpr, ExecPlan c
             aggregationNode.setHasNullableGenerateChild();
             aggregationNode.computeStatistics(optExpr.getStatistics());
 
+            // One phase aggregation prefer the inter-instance parallel to avoid local shuffle
+            if (node.isOnePhaseAgg()) {
```

Comment:
OnePhaseAgg has the same meaning as `aggregationNode.isNeedsFinalize() && hasNoExchangeNodes(inputFragment.getPlanRoot())`, so codes as follows can be refactored to be more succinct.